manifests/guestcluster/1.29/pvcsi.yaml

```diff
@@ -318,10 +321,20 @@ spec:
             - "--leader-election-lease-duration=120s"
             - "--leader-election-renew-deadline=60s"
             - "--leader-election-retry-period=30s"
+            - "--enable-capacity=true"
+            - "--capacity-ownerref-level=-1"
           imagePullPolicy: "IfNotPresent"
           env:
             - name: ADDRESS
               value: /csi/csi.sock
+            - name: NAMESPACE
```

Comment:
I think we do not need NAMESPACE and POD_NAME
cc: @akankshapanse

Reply:
I have removed it, as we are setting --capacity-ownerref-level=-1

Reply:
we need minimum of NAMESPACE variable to be provided for storage capacity to work in provisioner. 
Otherwise provisioner keeps crashing. Please add NAMESPACE variable back

Reply:
done added back NAMESPACE